apps/site/lib/site_web/templates/schedule/_shuttles.html.eex

```diff
@@ -10,6 +10,10 @@
 </script>
 
 <div class="row">
+  <div class="shuttles__alerts col-md-12">
+    <%= SiteWeb.AlertView.group(alerts: @alerts, route: @route, date_time: @date_time, priority_filter: :high) %>
```

Comment:
Does the `priority_filter` allow shuttle notifications to appear? Or is that going to be adjusted in a future commit anyway?

Reply:
It does, and it will be 🙂